src/stories/components/Review/Review.js

```diff
@@ -0,0 +1,148 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Fragment } from 'react';
+
+/**
+ * Component for Review element.
+ *
+ * @component
+ * @param {boolean} isUserVariant True if Review is User variant.
+ * @param {string} modifierClasses Class modifiers of the Review component.
+ * @param {string} title Title for the Review component.
+ * @param {number} rating Rating in stars of the Review component.
+ * @param {string} date Date of creation of the Review component.
+ * @param {string} name Name of user who wrote the review.
+ * @param {string} profUrl URL to profile of author of component.
+ * @param {string} content Content of the review component.
+ * @param {string} recipeTitle Title of the recipe referenced by component.
+ * @param {string} recipeLink Link to the recipe referenced by component.
+ * @return {object} (
+ *  <Review modifierClasses={modifierClasses} title={title}
+ *    rating={rating} date={date} name={name} profUrl={profUrl}
+ *    content={content} recipeTitle={recipeTitle} recipeLink={recipeLink}/>
+ * )
+ */
+export const Review = ({
+  isUserVariant, modifierClasses, title, rating, date, name,
+  profUrl, content, recipeTitle, recipeLink,
+}) => {
+  const classes = ['review', `${modifierClasses}`].join(' ').trim();
+
+  // Kind of hacky way to turn number of stars into array to use map on
+  const stars = [];
+  for (let i = 0; i < rating; i++) {
+    stars.push({ type: 'star', key: i });
+  }
```

Comment:
This is fine for now, while we don't know how the rating system is implemented on the API side. However, `map` would be a cleaner way to loop through an array.